tests/nlu/featurizers/test_count_vectors_featurizer.py

```diff
@@ -397,3 +400,46 @@ def test_count_vectors_featurizer_train():
     assert sen_vec is None
     assert (1, 1) == seq_vec.shape
     assert np.all(seq_vec.toarray()[0] == np.array([1]))
+
+
+@pytest.mark.parametrize(
+    "sentence, sequence_features, sentence_features, use_lemma",
+    [
+        ("go goes went go", [[1, 0, 0]], [[2, 1, 1]], False),
+        ("go goes went go", [[1]], [[4]], True),
+    ],
+)
+def test_count_vector_featurizer_use_lemma(
+    spacy_nlp: Any,
+    sentence: Text,
+    sequence_features: List[List[int]],
+    sentence_features: List[List[int]],
+    use_lemma: bool,
+):
+    ftr = CountVectorsFeaturizer({"use_lemma": use_lemma})
+
+    train_message = Message(sentence)
+    train_message.set(SPACY_DOCS[TEXT], spacy_nlp(sentence))
+    test_message = Message(sentence)
+    test_message.set(SPACY_DOCS[TEXT], spacy_nlp(sentence))
+
+    SpacyTokenizer().process(train_message)
+    SpacyTokenizer().process(test_message)
+
+    ftr.train(TrainingData([train_message]))
+
+    ftr.process(test_message)
+
+    seq_vecs, sen_vecs = test_message.get_sparse_features(TEXT, [])
+
+    assert isinstance(seq_vecs, scipy.sparse.coo_matrix)
+    assert isinstance(sen_vecs, scipy.sparse.coo_matrix)
+
+    actual_seq_vecs = seq_vecs.toarray()
+    actual_sen_vecs = sen_vecs.toarray()
+
+    print(actual_sen_vecs, sentence_features)
```

Comment:
what's the goal of printing that in a test? To have the logs in case something fails?

Reply:
oh, was just for me when debugging, will remove this